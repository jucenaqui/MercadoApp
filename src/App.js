import React from 'react';
import {Helmet} from "react-helmet";
import { AppRouter } from './router/AppRouter';

const App = () => {

    return (
      <>
       <Helmet>
            <meta charSet="UTF-8" />
            <title>MercadoApp</title>

            <link rel="icon" href=".%PUBLIC_URL%/favicon.ico" />
            <meta name="viewport" content="width=device-width, initial-scale=1" />
            <meta name="theme-color" content="#000000" />
            <meta
              name="description"
              content="Busca tus productos favoritos"
            />
            <link rel="manifest" href=".%PUBLIC_URL%/manifest.json" />
        </Helmet>
        <AppRouter />
      </>
  )
}

export default App;
