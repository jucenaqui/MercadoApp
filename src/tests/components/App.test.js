import { shallow } from "enzyme";
import App from "../../App";

describe('Test to <App />', () => {

    let wrapper = shallow(<App />);

    test('should show the component succesfull', () => {
        expect( wrapper ).toMatchSnapshot();
    });

    test('should have a AppRouter component', () => {
        expect( wrapper.find('AppRouter').exists() ).toBe(true);
    });

});
