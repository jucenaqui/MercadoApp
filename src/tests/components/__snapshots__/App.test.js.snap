// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to <App /> should show the component succesfull 1`] = `
<Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      charSet="UTF-8"
    />
    <title>
      MercadoApp
    </title>
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      content="#000000"
      name="theme-color"
    />
    <meta
      content="Busca tus productos favoritos"
      name="description"
    />
  </HelmetWrapper>
  <AppRouter />
</Fragment>
`;
